### PR TITLE
Update model accuracy

### PR DESCRIPTION
**Pull request by XuehaoSun**
## Type of Change

feature or bug fix or documentation or validation or others  
API changed or not

## Description

detail description

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Review by thuang6 — APPROVED** [review]

File: docs/source/validated_model_list.md [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
 1. [Validated Quantization Examples](#Validated-Quantization-Examples)
 
-    1.1. [TensorFlow Models with TensorFlow 2.15.0](#tensorflow-models-with-tensorflow-2150)
+   1.1. [TensorFlow Models with TensorFlow 2.16.1](#tensorflow-models-with-tensorflow-2161)
 
-    1.2. [PyTorch Models with Torch 2.2.1+cpu in PTQ Mode](#pytorch-models-with-torch-221cpu-in-ptq-mode)
+   1.2. [Keras Models with keras 2.15.1](#keras-models-with-keras-2151)
 
-    1.3. [PyTorch Models with Torch 2.2.1+cpu in QAT Mode](#pytorch-models-with-torch-221cpu-in-qat-mode)
+   1.2. [PyTorch Models with Torch 2.3.0+cpu in PTQ Mode](#pytorch-models-with-torch-230cpu-in-ptq-mode)
```
Comment (chensuyue) [inline docs/source/validated_model_list.md:11]: 1.3?

File: docs/source/validated_model_list.md [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
 
-    1.4. [PyTorch Models with Torch 2.0.1+cpu in WOQ Mode](#pytorch-models-with-torch-201cpu-in-woq-mode)
+   1.3. [PyTorch Models with Torch 2.3.0+cpu in QAT Mode](#pytorch-models-with-torch-230cpu-in-qat-mode)
```
Comment (chensuyue) [inline docs/source/validated_model_list.md:13]: 1.4

File: docs/source/validated_model_list.md [hunk 3]
```diff
@@ -1,21 +1,18 @@
...
 
-    1.5. [ONNX Models with ONNX Runtime 1.17.1](#onnx-models-with-onnx-runtime-1171)
-
-    1.6. [ONNX Models with ONNX Runtime 1.15.0 in WOQ Mode](#onnx-models-with-onnx-runtime-1150-in-woq-mode)
+   1.4. [ONNX Models with ONNX Runtime 1.18.1](#onnx-models-with-onnx-runtime-1181)
```
Comment (chensuyue) [inline docs/source/validated_model_list.md:15]: 1.5

File: docs/source/validated_model_list.md [hunk 4]
```diff
@@ -55,277 +52,297 @@ For more complete information about performance and benchmark results, visit www
   <tr>
     <td>ResNet50 v1.0</td>
     <td>pb</td>
-    <td>74.11%</td>
-    <td>74.27%</td>
-    <td>-0.22%</td>
-    <td>1720.00</td>
-    <td>582.18</td>
-    <td>2.95x</td>
+    <td>0.74112</td>
```
Comment (chensuyue) [inline docs/source/validated_model_list.md:55]: use the format `74.11%`

File: docs/source/validated_model_list.md [hunk 5]
```diff
@@ -55,277 +52,297 @@ For more complete information about performance and benchmark results, visit www
...
+    <td>0.74274</td>
+    <td>-0.00218</td>
+    <td>1732.923</td>
```
Comment (chensuyue) [inline docs/source/validated_model_list.md:58]: Keep two decimal places

File: docs/source/validated_model_list.md [hunk 6]
```diff
@@ -55,277 +52,297 @@ For more complete information about performance and benchmark results, visit www
...
+    <td>578.879</td>
+    <td>2.993584</td>
```
Comment (chensuyue) [inline docs/source/validated_model_list.md:60]: 2.99x

**Review by chensuyue — APPROVED** [review]

**Pull request merged**